instance/models/openedx_appserver.py

```diff
@@ -151,19 +151,6 @@ class Meta:
     )
 
     # Misc settings:
-    github_admin_organizations = JSONField(
-        max_length=256,
-        blank=True,
-        default=default_admin_organizations,
```

Comment:
@taranjeet @ahmedaljazzar If you remove this field, then the function `default_admin_organizations` is left alone (noone calls it), you should delete it too. Then you'll find that you'll need to delete a setting too (`DEFAULT_ADMIN_ORGANIZATION`). This setting is actually defined in our repo `ansible-secrets` and it's used by ansible to put it into the `.env` when deploying, and we'll need to delete it too (just to keep it synchronized). I can do that last step (it needs access) but we should remove the unused code here too.

Reply:
@clemente : I did remove that function and the related settings, but then the migrate crashes. You can read more about it [here|https://stackoverflow.com/a/40994166/2534102]

Reply:
We can replace this function in the migration files with `[]`. I think it's safe to do that as it's a data-migration that only expect a given result from some function Django doesn't know how it works. For previous migrations these have been already migrated and won't be affected, and for the new ones the value is just an empty list.

Reply:
I saw they are used in migrations… I don't know what will acceptable to Django. You can try what @ahmedaljazzar suggested. Noone should run this function in the future even if they migrate Ocim from 0 to the current migration, so it should be fine to delete it.